modules/listScore.js

```diff
@@ -0,0 +1,26 @@
+import plusScore from './addScores.js';
+
+const sumScores = () => {
+  const ScoresAll = document.getElementById('points');
+  ScoresAll.innerHTML = '';
+  const getScoresData = async () => {
+    const request = await fetch(
```

Comment:
- [ ] Please make sure that your design matches the wireframe 

| Your design | Wireframe |
| ---------------- | -------------- |
| ![image](https://user-images.githubusercontent.com/74744787/181821946-ad5bad95-c40c-46e5-8bb6-832b07caaf97.png) | ![image](https://user-images.githubusercontent.com/74744787/181822221-03b7eb3b-b94b-4242-b78d-1488bdd24094.png) |